app/addons/verifyinstall/tests/componentsSpec.js

```diff
@@ -0,0 +1,136 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
did not get it at first - but: is this file not a compilation result? and the next file is the actual one (i.e. this one shhould also not be checked in?)